docs/sphinx/source/index.rst

```diff
@@ -6,8 +6,227 @@
 RdTools Overview
 ================
 
-.. include:: readme_link.rst
+RdTools is an open-source library to support reproducible technical analysis of
+time series data from photovoltaic energy systems. The library aims to provide
+best practice analysis routines along with the building blocks for users to
+tailor their own analyses.
+In particular, PV production data is evaluated over several years to obtain
+rates of performance degradation and soiling loss. RdTools can handle
+both high frequency (hourly or better) or low frequency (daily, weekly,
+etc.) datasets. Best results are obtained with higher frequency data.
 
+Full examples are worked out in the example notebooks in the
+`example notebook`_.
+
+To report issues, contribute code, or suggest improvements to this
+documentation, visit the RdTools development repository on `github`_.
+
+Workflow
+--------
+
+RdTools supports a number of workflows, but a typical analysis follows
+the following:
+
+0. Import and preliminary calculations
+1. Normalize data using a performance metric
+2. Filter data that creates bias
+3. Aggregate data
+4. Analyze aggregated data to estimate the degradation rate and/or
+   soiling loss
+
+Steps 1 and 2 may be accomplished with the clearsky workflow (see the
+`example notebook`_) which can help eliminate problems from irradiance sensor
+drift.
+
+.. image:: _images/RdTools_workflows.png
+  :alt: RdTools workflow diagram
+
+Degradation Results
+-------------------
+
+The preferred method for degradation rate estimation is the year-on-year
+(YOY) approach, available in :py:func:`.degradation.degradation_year_on_year`.
+The YOY calculation yields in a distribution of degradation rates, the
+central tendency of which is the most representative of the true
+degradation. The width of the distribution provides information about
+the uncertainty in the estimate via a bootstrap calculation. The
+`example notebook`_ uses the output of
+:py:func:`.degradation.degradation_year_on_year` to visualize the calculation.
+
+.. image:: _images/Clearsky_result_updated.png
+   :alt: RdTools degradation results plot
+
+Two workflows are available for system performance ratio calculation,
+and illustrated in an example notebook. The sensor-based approach
+assumes that site irradiance and temperature sensors are calibrated and
+in good repair. Since this is not always the case, a 'clear-sky'
+workflow is provided that is based on modeled temperature and
+irradiance. Note that site irradiance data is still required to identify
+clear-sky conditions to be analyzed. In many cases, the 'clear-sky'
+analysis can identify conditions of instrument errors or irradiance
+sensor drift, such as in the above analysis.
```

Comment:
Suggest adding:

> The clear-sky analysis tends to provide less stable results than sensor-based analysis when details such as filtering are changed. We generally recommend that the clear-sky analysis be used as a check on the sensor based results, rather than as a stand-alone analysis.